### DISCUSSION TITLE
Functionally inverted icon

### DISCUSSION THREAD

**Discussion by developertoyosa** [dbeaver/dbeaver#19656] [Ideas]
### Description

The icons are inverted since visually the execution is interpreted differently, I have shared it with several friends and they have the same opinion

### DBeaver Version

Community Edition 23.0.1

### Operating System

Ubuntu 22.04

### Database and driver

![image](https://user-images.githubusercontent.com/56311711/230434845-9a51ef18-7da7-4fa5-badc-9e02bd946176.png)

### Steps to reproduce

1. Run a query

**Comment by E1izabeth** [comment]
The common icon for run action is such a triangle that we use. Both icons are dedicated to express running operation, but one of them is to run the whole script - script as a papyrus or paper with triange. I personally don't see any inversion here.
